tests/30rooms/13anonymousaccess.pl

```diff
@@ -297,3 +479,19 @@ sub matrix_set_room_history_visibility
       content => { history_visibility => $history_visibility }
    );
 }
+
+sub matrix_get_room_membership
+{
+   my ( $checking_user, $room_id, $tested_user ) = @_;
+
+   matrix_get_room_state( $checking_user, $room_id,
+      type => "m.room.member",
+      state_key => $tested_user->user_id,
+   )->then( sub {
```

Comment:
In the two-argument version of `->then` I usually notate this as

    ->then(
       sub { ... },
       sub { ... },
    )

With each `sub` sitting on its own line, so it looks Obviously Different to the normal single-arg `->then` chaining; and thus calls attention to it on skimming. E.g. compare to

  https://github.com/matrix-org/sytest/blob/develop/tests/00expect_http_fail.pl#L10